tests/test_display_fixes.py

```
@@ -0,0 +1,125 @@
+"""
+Tests for display formatting fixes.
+"""
+
+import pytest
```

Ruff (0.12.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)